src/components/Reservation.js

```diff
@@ -1,9 +1,187 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { Navigate } from 'react-router-dom';
+import moment from 'moment';
+import { Form, Button } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import styles from '../styles/reservation.module.css';
+import postReservationThunk from '../helpers/postReservation';
+
+export default function Reservation(props) {
+  const [startDate, setStartDate] = useState(null);
+  const [endDate, setEndDate] = useState(null);
+  const [submitted, setSubmitted] = useState(false);
+  let { destination } = props;
+  const destinationPassed = destination !== 'default';
+  destination = destination === 'default' ? 'Bora Bora' : destination;
+  const [chosenDestination, setChosenDestination] = useState(destination);
+  const dispatch = useDispatch();
+
+  const username = useSelector((state) => state.user.username);
+  const token = useSelector((state) => state.user.token);
+
+  const destinations = useSelector((state) => state.destinations.destination);
+  destinations.forEach((des, idx) => ({ ...des, id: idx }));
+  // const destinations = [
+  //   {
+  //     id: 1,
+  //     name: 'Bora Bora',
+  //     location: 'French Polynesia',
+  //     image_url: 'https://res.cloudinary.com/dqdrsilxz/image/upload/v1636638534/bora-bora-island_tcyoev.jpg',
+  //     price_per_day: '150',
+  //   },
+  //   {
+  //     id: 2,
+  //     name: 'Rome',
+  //     location: 'Italy',
+  //     image_url: 'https://res.cloudinary.com/dqdrsilxz/image/upload/v1636638500/Rome-Italy_isahvk.jpg',
+  //     price_per_day: '100',
+  //   },
+  //   {
+  //     id: 3,
+  //     name: 'Gracier National Park',
+  //     location: 'USA',
+  //     image_url: 'https://res.cloudinary.com/dqdrsilxz/image/upload/v1636573679/sample.jpg',
+  //     price_per_day: '100',
+  //   },
+  // ];
+
+  const setBackgroundImg = () => {
+    const url = destinations.filter((place) => place.name === chosenDestination);
+    return url[0].image_url;
+  };
+
```

Comment:
Hey Yen there is some issue with this peace of code. I can't render the form.
![image](https://user-images.githubusercontent.com/63932912/142597001-f17176b7-6880-4677-b65f-7ab9f4217fbe.png)

Reply:
I've commented out the part for changing background. You can check it again.